src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
+const spanList = document.getElementsByClassName('population');
+let summ = 0;
+let avg = 0;
+
+if (spanList.length > 0) {
+  summ = [...spanList].reduce(
+    (accumulator, currentValue) =>
+      accumulator + parseInt(currentValue.textContent.replaceAll(',', '')),
+    0
+  );
+
+  avg = summ / spanList.length;
```

Comment:
I would recommend to follow `single responsibility principle`
Create one variable where you will find a sum
And another variable where you will get an average

Reply:
and I did it